timemachine/cpp/src/neighborlist.cu

```diff
@@ -76,30 +76,23 @@ void Neighborlist<RealType>::compute_block_bounds_host(
 
     assert(N == N_);
     assert(D == 3);
-    int h_rebuild = 1;
 
     double *d_coords = gpuErrchkCudaMallocAndCopy(h_coords, N*3*sizeof(double));
-    double *d_box = gpuErrchkCudaMallocAndCopy(h_box, 3*3*sizeof(double));
-    int *d_rebuild = gpuErrchkCudaMallocAndCopy(&h_rebuild, 1*sizeof(int));
-    cudaStream_t stream = static_cast<cudaStream_t>(0);
+    double *d_box = gpuErrchkCudaMallocAndCopy(h_box, 3*3*sizeof(double));  
+   
     this->compute_block_bounds_device(
         N,
         D,
         d_coords,
         d_box,
-        d_rebuild,
-        stream
+        static_cast<cudaStream_t>(0)
     );
-
-    cudaStreamSynchronize(stream);
+    // Does this need to peek at the last error?
```

Comment:
Good question - probably wouldn't hurt to peak after the `cudaDeviceSynchronize()`?